src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/ConnectionPoolTest/ConnectionPoolTest.cs

```diff
@@ -178,20 +178,23 @@ private static void ReclaimEmancipatedOnOpenTest(string connectionString)
             string newConnectionString = (new SqlConnectionStringBuilder(connectionString) { MaxPoolSize = 1 }).ConnectionString;
             SqlConnection.ClearAllPools();
 
-            InternalConnectionWrapper internalConnection = CreateEmancipatedConnection(newConnectionString);
+            using var outerConnection = new SqlConnection(newConnectionString);
```

Comment:
Please revert the changes in this `ReclaimEmancipatedOnOpenTest` test because it is designed to trigger the reclaim behavior without explicitly close the connection on purpose. Otherwise, the new connection won't be reclaimed.

Reply:
Done.